packages/react-instantsearch-hooks-server/test/is-a-module.mjs

```diff
@@ -0,0 +1,5 @@
+import assert from 'assert';
+
+import * as ReactInstantSearchHooksServer from 'react-instantsearch-hooks-server';
+
+assert.ok(ReactInstantSearchHooksServer);
```

Comment:
So that we know why it fails (if it does):

```suggestion
assert.ok(ReactInstantSearchHooksServer, 'The module `react-instantsearch-hooks-server` is not ESM compliant.');
```

Reply:
yes, can change, although if it's not esm compliant it will throw an error before this assert, this assert is to make sure it's not empty

Reply:
Oh I see. Ignore my comment then.